packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -235,7 +235,7 @@ export const TimePickerButtonLabel = memo<LabelProps>(({ hideText, value, timeZo
   }
 
   return (
-    <span className={styles.container}>
+    <span className={styles.container} aria-live="polite" aria-atomic="true">
```

Comment:
I feel `assertive` should be better here, as this update is usually an immediate consequence of a button press (or whatever other method you use to update it). It feels a bit sluggish with the delay, and might be harder to associate with the key press.

Reply:
🤔 According to the following, I'd go for `aria-live="polite"` as I don't think an update on this is critical 🤷‍♀️ 

> `aria-live="assertive"` should only be used for time-sensitive/critical notifications that absolutely require the user's immediate attention.

Reply:
But it is time sensitive, because it is generated by the user action.

Reply:
I think polite works fine with previous and next arrows in this case, but with the zoom out button it is a bit annoying.